cumulusci/tasks/datadictionary.py

```diff
@@ -0,0 +1,304 @@
+import csv
+import io
+import os
+import xml.etree.ElementTree as ET
+import zipfile
+
+from collections import defaultdict
+from cumulusci.tasks.github.base import BaseGithubTask
+from distutils.version import LooseVersion
+
+
+class GenerateDataDictionary(BaseGithubTask):
+    task_docs = """
+    Generate a data dictionary for the project by walking all GitHub releases.
+    The data dictionary is output as two CSV files.
+    One, in `object_path`, includes the Object Name, Object Label, and Version Introduced,
+    with one row per packaged object.
+    The other, in `field_path`, includes Object Name, Field Name, Field Label, Field Type,
+    Picklist Values (if any), Version Introduced.
+    """
+
+    task_options = {
+        "object_path": {
+            "description": "Path to a CSV file to contain an sObject-level data dictionary.",
+            "default": "sObject Data Dictionary.csv",
+        },
+        "field_path": {
+            "description": "Path to a CSV file to contain an field-level data dictionary.",
+            "default": "Field Data Dictionary.csv",
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        self.options["object_path"] = (
+            self.options.get("object_path") or "sObject Data Dictionary.csv"
```

Comment:
It looks like the default filenames are used more than once. Could we define the default filenames as constant class members and then refer to them here and where the defaults are defined in `task_options`?

Reply:
@Br4nd0R the defaults in task_options aren't actually used for anything at this time.

Reply:
Even if they aren't used and we have two explicitly defined strings with the same value, wouldn't it make sense to create a variable and reference that instead of defining the same string twice?